src/components/document/TableView/TableView.jsx

```diff
@@ -1,6 +1,78 @@
-const TableView = () => { 
-    return <div>
-        TableView
+import CheckIcon from "@/assets/check_fill.svg";
+import CloseIcon from "@/assets/close_fill.svg";
+import CommentIcon from "@/assets/comment_fill.svg";
+import NavigateToIcon from "@/assets/navigate_to_icon.svg";
+import { Slider } from 'antd';
+import { useState } from "react";
+
+const RedactionFilterDropdownMenu = () => {
+    //TODO: Some sort of dropdown
+    return <div className="w-full text-end">
+        Accepted Suggestions 
+    </div>
+}
+
+const splitText = (document) => {
+    return document.body.children.map((paragraph) => {
+        return paragraph.children
+    }).flat(1)
+}
+
+const TableEntry = (sentence) => {
+    const text = sentence.sentence.text
+    // Temporarily insert " REDACTED " into the sentence
+    const sentenceWithRedactedWord = text.slice(0, text.length/2) + " REDACTED " + text.slice(text.length/2)
```

Comment:
For whatever it's worth, you can insert into slate at a given location with [this api](https://docs.slatejs.org/concepts/04-transforms#the-at-option). You need to import the editor though. Not important immediately since this is a temporary solution anyways, just a heads up

Reply:
But I do think we'll have to change up the sentence splitting code a decent amount once Slate is in. Maybe pull this functionality into a separate function so it's easier to sub out later?

Reply:
Sure I'll refactor this into a separate function